en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,82 @@
+# PBKDF2 hashing
+Password-Based Key Derivation Function 2 (PBKDF2) hashing algorithm is a modern hashing algorithm recommended by NIST.  We can use PBKDF2 hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
+
+This guide walks you through the steps of configuring PBKDF2 as the hashing algorithm of a JDBC userstore.
+
+!!! note
+    Currently, PBKDF2 supports only JDBC userstores of WSO2 Identity Server.
+
+## Configure PBKDF2 hashing
+To configure PBKDF2 hashing on a JDBC user store:
+
+1. On Identity Server management console (`https://<IS_HOST>:<PORT>`) and [create a JDBC user store](../configure-secondary-user-stores).
+
+    !!! info
+        - You may also use an existing user store which does not have any users in it. If you already have users in the userstore, once the hashing algorithm is configured these users will not be able to get authenticated.
+        - PBKDF2 is also supported by [primary JDBC userstores](../configure-a-jdbc-user-store), but PBKDF2 should be enabled before the initial server startup by adding the following to the `deployment.toml` file.
```

Comment:
Shall we highlight this? `should be enabled before the initial server startup`

Reply:
```suggestion
        - PBKDF2 is also supported by [primary JDBC userstores](../configure-a-jdbc-user-store), but PBKDF2 **should be enabled before the initial server startup** by adding the following to the `deployment.toml` file.
```